tests/component_handlers/helm_wrapper/test_helm_wrapper.py

```diff
@@ -457,17 +457,22 @@ def test_should_call_run_command_method_when_helm_template_without_optional_args
         )
 
     @pytest.mark.parametrize(
-        "version",
+        "version_object, version",
```

Comment:
consider swapping those (as input, output in that order) and give them names where it's more clear what they're used for, e.g. `raw_version`, `expected_version` or similar

Reply:
done